### PR TITLE
Admin verb to change damage

### PR DESCRIPTION
**Pull request by xxalpha**
When you use the verb as an admin.

### CODE REVIEW

**Comment by optimumtact** [comment]
Don't open an pr like this without announcing it to the game.

Redo it.

File: code/__HELPERS/game.dm [hunk 1]
```diff
@@ -454,3 +454,11 @@
 	new_character.key = G_found.key
 
 	return new_character
+
+/proc/damage_buff(duration, client/C)
+	if(duration <= 0 || !istype(C))
+		return
+	C.SDQL2_query("UPDATE /obj/item/weapon IN world SET force = 100, throwforce = 100")
+	sleep(duration)
```
Comment (optimumtact) [inline code/__HELPERS/game.dm]: Use check_tick instead

Comment (xxalpha) [inline code/__HELPERS/game.dm]: why though

**Comment by KorPhaeron** [comment]
Does this button have any real purpose beyond periodically making a round unplayable when an admin gets bored?

File: code/__HELPERS/game.dm [hunk 2]
```diff
@@ -454,3 +454,11 @@
 	new_character.key = G_found.key
 
 	return new_character
+
+/proc/damage_buff(duration, client/C)
+	if(duration <= 0 || !istype(C))
+		return
+	C.SDQL2_query("UPDATE /obj/item/weapon IN world SET force = 100, throwforce = 100")
```
Comment (Jordie0608) [inline code/__HELPERS/game.dm]: Why not have the force admin-input too?

**Comment by KorPhaeron** [comment]
Gonna shill my own work again and say that https://github.com/tgstation/tgstation/pull/16336

Would let admins do the stupid stuff in this PR, but also serve legitimate purposes in being able to tweak lethality

**Comment by Jordie0608** [comment]
This could, like a bunch of buttons, be easily accomplished by just running the SDQL2 query.

**Comment by optimumtact** [comment]
haha that implies the admins know how to use SDQL2 though :grin:

**Comment by xxalpha** [comment]
Here we go again with the argument of "why automate things when admins can do them manually, code lines are expensive and we must use them sparely".

If maintainers decide the other PR does things better they can close this one.

**Comment by optimumtact** [comment]
Actually if you look closely Kor was making a comment from a headmin managing admins perspective and Jordie was simply commenting that admins can already do this.

**Comment by Jordie0608** [comment]
Just replace all buttons with a readme on how to use Advanced Proc Call.

**Comment by xxalpha** [comment]
And SDQL2

**Comment by Jordie0608** [comment]
?

**Comment by optimumtact** [comment]
Cucked man cucked

**Comment by xxalpha** [comment]
If admins aren't allowed to press buttons I see no reason for this to exist.

**Comment by Jordie0608** [comment]
That was a joke, I've no worthwhile problem with this.

**Pull request closed without merging**